### DISCUSSION TITLE
When request times out, no Log is created

### DISCUSSION THREAD

**Discussion by darrensheffield** [Yue-Yang/ChatGPT-Siri#72] [General]
Story

- When a long input is sent on the first interaction, and
- OpenAI takes longer to respond than expected
- The Shortcut produces a timeout error and no Log is created in iCloud Drive

**Comment by Yue-Yang** [comment]
Yes, It's possible to get this. But I can't setup a longer timeout time. Gpt3.5 is faster than gpt4

**Comment by eiko4** [comment]
@Yue-Yang use another http post method via Scriptable/any another app

https://github.com/sindresorhus/Actions/issues/44#issuecomment-1779841313
